go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/grpc-ecosystem/go-grpc-middleware/v2
 go 1.14
 
 require (
-	github.com/gogo/protobuf v1.2.1
-	github.com/golang/protobuf v1.3.2
+	github.com/gogo/protobuf v1.3.1
```

Comment:
Nothing imports but still this guy is keeping coming back here ;p

Reply:
Try `go mod why github.com/gogo/protobuf`

Reply:
It says "nothing is using it" ;p

Anyway, found, updating PR